### PR TITLE
docs(kit): OSim KIT for 24.03 Release

### PR DESCRIPTION
**Pull request by dominikoeh**
## Description

This contribution contains updates to the most recent standard releases.

- Adoption View
  - Business process for scenario related data exchange
  - MaterialFlowScenarioRequest aspect model
- Development View:
  - requestScenarioFeedback API
  - receiveScenarioFeedback API
  - pushScenarioState API

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by theodorisinger** [comment]
I'm the contact in OSim Team

File: docs-kits/kits/OSim Kit/page_adoption-view.md [hunk 1]
```diff
@@ -125,3 +163,88 @@ The following appropriate configurations must be made in advance of the data exc
...
+ "odrl:operator": {
+ "@id": "odrl:eq"
+ },
+ "odrl:rightOperand": "active"
+}
+```
+
+## Standards
+
+Our relevant standards can be downloaded from the official [Catena-X Standard Library](https://catena-x.net/de/standard-library).
```
Comment (maximilianong) [inline docs-kits/kits/OSim Kit/page_adoption-view.md:242]: Can you name the concrete standards that are relevant for your KIT + the version.

**Review by maximilianong — CHANGES_REQUESTED** [review]
In general really good PR and update of the KIT!
Thanks.

Just one small change regarding the standards required.

File: docs-kits/kits/OSim Kit/page_adoption-view.md (hunk 1 above)
Comment (dominikoeh) [reply docs-kits/kits/OSim Kit/page_adoption-view.md]: Links to the already released standards have been added.

The new standard, which has been approved, but is yet to be released has been mentioned with a link to the library

**Review by maximilianong — APPROVED** [review]
Ready for release 24.03.
Thanks to @dominikoeh and team.

**Review by danielmiehle — APPROVED** [review]

**Pull request merged**